promql/quantile.go

```diff
@@ -181,11 +186,12 @@ func ensureMonotonic(buckets buckets) {
 // quantile calculates the given quantile of a vector of samples.
 //
 // The Vector will be sorted.
-// If 'values' has zero elements, NaN is returned.
+// If 'values' has zero elements
+// If q==NaN, NaN is returned.
```

Comment:
An "or" is missing, and the 2nd "If" needs to be lower-case. (Or Add ", NaN is returned." at the end of line 189.)